docs/docs/features/environments.mdx

```diff
@@ -30,8 +36,24 @@ the framework you're using, some environment variables are not exposed by defaul
 appropriate prefix, e.g. `NEXT_GROWTHBOOK_CLIENT_KEY='sdk-abc123'` in order to access environment variables in Next.js
 client code.
 
+## Environment Inheritance
+
+When creating an environment via the API, you can specify a `parent` environment it should inherit from.
+The newly created environment will share the same feature rules as its parent for all features that exist
+at the time of creation. Any changes to feature rules in either the parent or the new environment will
+not be shared and will cause the two environments to diverge. In other words, the new environment is
+a clone or fork of the parent.
+
 :::note
 
-It's possible for a feature to be enabled for an environment and still be considered "off". This happens when its value is set to `false`, `null`, `0`, or an empty string.
+**Environment Inheritance** is a GrowthBook Enterprise feature.
 
 :::
+
+There are two main usages for Environment Inheritance:
```

Comment:
Possible edit:
**Common Use Cases**
- Adding new environments to an existing setup (e.g., adding "staging" after having "dev" and "production")
- Troubleshooting production issues by creating an isolated test environment to safely diagnose problems